docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<ol><li>There seems to be an extra space at the beginning of the title " Novel Time Series...". Please remove the space</li><li>"matrix profile" are two separate words, not one. Please add a space</li><li><code>If a dataset has two similar subsequences that are far from the rest of subsequences</code> Can you please elaborate on this sentence? I am not getting your point. If they are similar enough then they may not be the top motif but it is possible to be, say, in the top 50 motifs?</li><li>Usually, in the opening paragraph, we want to try an explain what the problem is that contrast profiles can help solve. Can you clearly explain the problem? It is okay to borrow from the published paper</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>